src/User-repo.js

```diff
@@ -47,8 +85,9 @@ class UserRepo {
         objectSoFar[dataItem.userID].push(dataItem[relevantData])
       }
       return objectSoFar;
-    }, {});
+    }, {})
```

Comment:
The issue with the way they did this is that It just makes smaller and less clear arrays of objects from what we already have